src/main/resources/application.yml

```diff
@@ -262,7 +261,7 @@ quarkus:
   jackson:
     fail-on-unknown-properties: true
   native:
-    additional-build-args: --initialize-at-run-time=io.confluent.idesidecar.restapi.auth.CCloudOAuthConfig\,io.confluent.idesidecar.restapi.auth.CCloudOAuthContext\,io.confluent.ide.sidecar.restapi.util.WebClientFactory\,org.apache.kafka.common.security.authenticator.SaslClientAuthenticator\,org.apache.avro.file.DataFileWriter\,io.confluent.kafka.schemaregistry.avro.AvroSchemaUtils\,com.github.luben.zstd\,org.xerial.snappy\,io.confluent.idesidecar.restapi.application.SnappyNativeLoader
+    additional-build-args: --initialize-at-run-time=io.confluent.idesidecar.restapi.auth.CCloudOAuthConfig\,io.confluent.idesidecar.restapi.auth.CCloudOAuthContext\,io.confluent.ide.sidecar.restapi.util.WebClientFactory\,org.apache.kafka.common.security.authenticator.SaslClientAuthenticator\,org.apache.avro.file.DataFileWriter\,io.confluent.kafka.schemaregistry.avro.AvroSchemaUtils\,com.github.luben.zstd\,org.xerial.snappy\,io.confluent.idesidecar.restapi.application.SnappyNativeLoader\,io.confluent.idesidecar.restapi.models.graph.RealCCloudFetcher\,io.confluent.idesidecar.websocket.proxy.ProxyContext\,io.confluent.idesidecar.restapi.proxy.FlinkDataPlaneProxyProcessor
```

Comment:
We need to initialize classes that read env-related config options at runtime, so that the extension can set the config option `ide-sidecar.connections.ccloud.base-path` at runtime.